crates/sui-json-rpc/src/api.rs

```diff
@@ -663,6 +664,20 @@ pub trait EventReadApi {
     ) -> RpcResult<EventPage>;
 }
 
+#[open_rpc(namespace = "sui", tag = "Threshold BLS APIs")]
```

Comment:
lets add this module to openrpc schema file
https://github.com/MystenLabs/sui/blob/2a3d2853d64f539494869aac5baf0670962bf1fd/crates/sui-open-rpc/src/generate_json_rpc_spec.rs#L59